SI/Sem.02/Pract.02/Task8/Task8.cpp

```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+   int num;
+   cout << "Arabic numeral: ";
+   cin >> num;
+   
+   cout << "Roman numeral: ";
+   for (size_t i = 0; i < num/10; i++)
```

Comment:
The program should work only for ONE DIGIT numbers.
Also, check your case for the following input: 4.